step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManager.java

```diff
@@ -160,6 +167,16 @@ private void deleteAutomationPackageEntities(AutomationPackage automationPackage
         deletePlans(automationPackage);
         deleteSchedules(automationPackage);
         deleteResources(automationPackage);
+        deleteAlertingRules(automationPackage);
```

Comment:
In a second iteration we should do all this in a pluggable way i.e. the AutomationPackageManager should expose the hooks to which the different managers should register to

Reply:
@jeromecomte 
Fully agree. I think the pluggable implementation should cover both `AutomationPackageManager` and the `YamlAutomationPackageSchemaGenerator`.